acceptance/services_test.go

```diff
@@ -78,6 +81,187 @@ var _ = Describe("Services", func() {
+			deleteServiceFromNamespace(namespace2, service2)
+
+			env.DeleteNamespace(namespace1)
+			env.DeleteNamespace(namespace2)
+
+			// Remove transient settings
+			out, err := proc.Run("", false, "rm", "-f", tmpSettingsPath)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			env.DeleteEpinioUser(user1)
+			env.DeleteEpinioUser(user2)
+		})
+
+		It("list all services", func() {
+			By("create them in different namespaces")
+			// create service1 in namespace1
+			env.TargetNamespace(namespace1)
+			out, err := env.Epinio("", "service", "create", "mysql-dev", service1)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			// create service2 in namespace2
+			env.TargetNamespace(namespace2)
+			out, err = env.Epinio("", "service", "create", "mysql-dev", service2)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			// show all the services (we are admin, good to go)
+			By("show it")
+			out, err = env.Epinio("", "service", "list", "--all")
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			Expect(out).To(MatchRegexp("Listing all Services"))
+
+			Expect(out).To(MatchRegexp(regex.TableRow(namespace1, service1, "mysql-dev", "not-ready")))
+			Expect(out).To(MatchRegexp(regex.TableRow(namespace2, service2, "mysql-dev", "not-ready")))
+
+			By("wait for deployment")
+			Eventually(func() string {
+				out, _ := env.Epinio("", "service", "list", "--all")
+				return out
+			}, "2m", "5s").Should(MatchRegexp(regex.TableRow(namespace1, service1, "mysql-dev", "deployed")))
+
+			Eventually(func() string {
+				out, _ := env.Epinio("", "service", "list", "--all")
+				return out
+			}, "2m", "5s").Should(MatchRegexp(regex.TableRow(namespace2, service2, "mysql-dev", "deployed")))
+
+			By(fmt.Sprintf("%s/%s up", namespace1, service1))
+		})
+
+		It("list only the services in the user namespace", func() {
+			By("create them in different namespaces")
+
+			// impersonate user1 and target namespace1
+			updateSettings(user1, password1, namespace1)
+
+			// create service1 in namespace1
+			out, err := env.Epinio("", "service", "create", "mysql-dev", service1, "--settings-file", tmpSettingsPath)
+			Expect(err).ToNot(HaveOccurred(), out, tmpSettingsPath)
+
+			// impersonate user2
+			updateSettings(user2, password2, namespace2)
+
+			// create service2 in namespace2
+			env.TargetNamespace(namespace2)
+			out, err = env.Epinio("", "service", "create", "mysql-dev", service2, "--settings-file", tmpSettingsPath)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			// show only owned namespaces (we are user2, only namespace2)
+			By("show it")
+			out, err = env.Epinio("", "service", "list", "--all", "--settings-file", tmpSettingsPath)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			Expect(out).To(MatchRegexp("Listing all Services"))
+
+			Expect(out).NotTo(MatchRegexp(regex.TableRow(namespace1, service1, "mysql-dev", "not-ready")))
+			Expect(out).To(MatchRegexp(regex.TableRow(namespace2, service2, "mysql-dev", "not-ready")))
```

Comment:
Same here wrt timestamps.

Reply:
Confirmed that the new `--all` code in the cli is missing printing of the timestamps.